modules/importer/swa-importer.js

```diff
@@ -426,6 +426,7 @@ export default class SWAImporter extends FormApplication {
                     const ffgSkill = Object.keys(skills).find((s) => skill.toLowerCase() === s.toLowerCase());
 
                     if (ffgSkill) {
+                      adversary.data.skills[ffgSkill].groupskill = true;
```

Comment:
Testing this out discovered two things,
1. Setting groupskill = true on a rival ignores their skill ranks
2. This never gets hit for minions because their .skills attribute is an array instead of a map

Reply:
Addressed both of these issues with latest changes